src/better_boundaries/boundary_conditions.cpp

```diff
@@ -0,0 +1,272 @@
+//========================================================================================
```

Comment:
This file should be removed. It's an artifact of when you based your branch. This isn't in master anymore.